### PR TITLE
Update Design.md

### PR DESCRIPTION
**Pull request by daniellcurto**

### CODE REVIEW

**Review by MMercer8989 — CHANGES_REQUESTED** [review]
I need you to make one change to this before I can approve it, you did not include the closing pipe on line 25 and thus the table will not be formatted correctly when viewed.

**Review by MMercer8989 — APPROVED** [review]
thank you for correcting the formatting issue

**Review by mryindles — APPROVED** [review]
Thank you for making the changes Matt requested, and keeping things in line with the proper formatting, looks good, you have my approval.

**Pull request merged**
